mono/btls/btls-util.c

```diff
@@ -27,12 +27,11 @@ mono_btls_util_asn1_time_to_ticks (ASN1_TIME *time)
 	ASN1_GENERALIZEDTIME *gtime;
 	struct tm tm;
 	int64_t epoch;
-	int ret;
 	
 	memset (&tm, 0, sizeof (tm));
 
 	gtime = ASN1_TIME_to_generalizedtime (time, NULL);
-	ret = asn1_generalizedtime_to_tm (&tm, gtime);
+	asn1_generalizedtime_to_tm (&tm, gtime);
```

Comment:
@baulig I will do that in a follow-up PR as @lewurm  [suggested](https://github.com/mono/mono/pull/5064#pullrequestreview-45960304) - as soon as this one got merged! I would change the comment to `/* FIXME: check in managed code */` then - is that ok?